src/clientsettingsdlg.cpp

```diff
@@ -229,13 +229,15 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
                                          "let you choose input or outputs of your device(s). "
                                          "More information can be found on jamulus.io." );
 
-    QString strSndCardDriverSetupTT = tr ( "Opens the driver settings. Note: " ) + APP_NAME +
-                                      tr ( " currently only supports devices supporting a sample rate of " ) +
-                                      QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) +
-                                      tr ( "Hz. "
+    QString strSndCardDriverSetupTT = tr ( "Opens the driver settings. Note: %1 "
+                                           "currently only supports devices supporting a sample rate of %2"
```

Comment:
Is "works" clearer than merely "supports" (it removes the slighty awkward repetition of "support" too)?

```suggestion
                                           "currently only works with devices supporting a sample rate of %2"
```